packages/php-wasm/universal/src/lib/sandboxed-spawn-handler-factory.ts

```diff
@@ -62,7 +62,7 @@ export function sandboxedSpawnHandlerFactory(
 			return;
```

Comment:
@Utsav-Ladani we got [some changes](https://github.com/WordPress/wordpress-playground/commit/d07a2ef49f12dfde55faf8bdb525e6e1d94f9a8b) on origin. Let's rebase and see how it goes.